docs/guides/routing-traffic-to-your-app.md

```diff
@@ -25,8 +25,8 @@ With this architecture, the coffee application is not accessible outside the clu
 on the hostname `cafe.example.com` so that clients outside the cluster can access it.
 
 To do this, we will install NGINX Gateway Fabric and create two Gateway API resources:
-a [Gateway](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1beta1.Gateway) and
-an [HTTPRoute](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPRoute).
+a [Gateway](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1.Gateway) and
```

Comment:
this was before, but shall we remove v1alpha2 from the uri in both links? and in other similar links in this doc?